test/test-cases/functional/saic/test_vm_to_vm_commn_acl_inbound.py

```diff
@@ -0,0 +1,179 @@
+import json
+from pathlib import Path
+from pprint import pprint
+import time
+import pytest
+import sys
+sys.path.append("../utils")
+import snappi_utils as su
+
+current_file_dir = Path(__file__).parent
+
+"""
+This covers following scenario :
```

Comment:
Could you please extend test description here and in two other test modules. It's difficult to understand what is expected when part of the configuration is in a separate file and another part is here but "smeared" in the test function code.
I would suggest adding a function kind of build_vxlan_packet (or flow) which could be used here and which can help reduce number of code lines in the test cases.
In that case it will be easier to comprehend what is sent and what is expected.

Reply:
Modified